lib/utils/tls.go

```diff
@@ -54,6 +62,74 @@ type CertificateName struct {
 	OrganizationalUnit []string `json:"org_unit"`
 }
 
+// TLSCredentials keeps the typical 3 components of a proper HTTPS configuration
+type TLSCredentials struct {
```

Comment:
Let's just reuse this struct from Teleport codebase. I.e. copy/update the method, but we don't have to copy this struct, just use Teleport's one.